src/kibana/directives/auto_select_if_only_one.js

```diff
@@ -1,12 +1,16 @@
 define(function (require) {
   var module = require('modules').get('kibana');
 
-  module.directive('autoSelectIfOnlyOne', function () {
+  module.directive('autoSelectIfOnlyOne', function ($parse) {
     return {
       restrict: 'A',
       require: 'ngModel',
+      scope: {
+        autoSelectIfOnlyOne: '&autoSelectIfOnlyOne'
+      },
```

Comment:
I'm just curious as to why you decided to use isolate scope instead of something like `$parse`. (Oh, also, it looks like there's an unused reference to `$parse` above.) I've been under the understanding that, because each element can only have one isolate scope, it's best to avoid using isolate scope unless the directive is basically the "main" directive for a given element.